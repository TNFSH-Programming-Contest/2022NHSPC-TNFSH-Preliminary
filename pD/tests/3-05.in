6 7
3 1 2 2 0 3
2 1 2 1 2 1 2
.T...T.
T.T....
.T..T.T
.......
.T...TT
..T....
