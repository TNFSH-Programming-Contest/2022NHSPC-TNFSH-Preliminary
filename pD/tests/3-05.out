ATA.AT.
T.T...A
ATA.T.T
....A.A
.T...TT
.ATA.A.
